ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-hcp-4.14-stable-x86-loaded-upgrade-from-4.14.yaml

```diff
@@ -0,0 +1,52 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-loaded-upgrade-414z
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: perfscale-rosa-multiaz-6nodes-stage-loaded-upgrade414
+  cron: 0 8 * * 3
```

Comment:
Can you update the Job names to match the existing one's like this:
`perfscale-rosa-6nodes-loaded-upgrade-413to414`
which is similar to what we have adhered here
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86-loaded-upgrade-from-4.14.yaml#L32C7-L32C52

Reply:
Fixed in new code, thanks for your suggestion!